src/CSET/operators/plot.py

```diff
@@ -458,8 +458,10 @@ def _plot_and_save_postage_stamp_spatial_plot(
 
     # Put the shared colorbar in its own axes.
     colorbar_axes = fig.add_axes([0.15, 0.07, 0.7, 0.03])
-    colorbar = fig.colorbar(plot, colorbar_axes, orientation="horizontal")
-    colorbar.set_label(f"{cube.name()} / {cube.units}")
+    colorbar = fig.colorbar(
+        plot, colorbar_axes, orientation="horizontal", pad=0.08, shrink=0.75
```

Comment:
```suggestion
        plot, colorbar_axes, orientation="horizontal", pad=0.042, shrink=0.75
```
I (and James) prefer to maximise the plot size with really tight padding. The value 0.042 was picked via a bit of experimentation, producing the following:

![untitled_462147 0](https://github.com/user-attachments/assets/2a2290c6-101b-46ac-9117-c329f96f1614)

Reply:
Happy with suggestion - it would be tempting to update shrink=0.7 to reduce colorbar extent closer to width of UK plot?